Makefile

```diff
@@ -73,6 +73,9 @@ db: vendor wait-for-db
 db-diff: vendor wait-for-db                                                                            ## Generate a migration by comparing your current database to your mapping information
 	$(EXEC) $(CONSOLE) doctrine:migration:diff
 
+db-dump: vendor wait-for-db                                                                            ## Generate a migration by comparing your current database to your mapping information
```

Comment:
The name is misleading, I would go for `db-diff-dump`.

Please update the comment.